python-sdk/src/astro/databases/snowflake.py

```diff
@@ -579,17 +566,26 @@ def load_pandas_dataframe_to_table(
         :param if_exists: Strategy to be used in case the target table already exists.
         :param chunk_size: Specify the number of rows in each batch to be written at a time.
         """
-        if if_exists == "replace":
-            self.create_table(target_table, dataframe=source_dataframe)
+
+        auto_create_table = False
+        if if_exists == "replace" or not self.table_exists(target_table):
+            auto_create_table = True
+
+        if target_table.metadata.schema:
+            target_table.metadata.schema = target_table.metadata.schema.upper()
+
+        if target_table.metadata.database:
+            target_table.metadata.database = target_table.metadata.database.upper()
```

Comment:
Are we happy changing the schema and database capitalization to upper?

Reply:
Was not needed, removed it.